src/wasm/asm-wasm-builder.cc

```diff
@@ -524,6 +527,7 @@ class AsmWasmBuilderImpl : public AstVisitor {
   Isolate* isolate_;
   Zone* zone_;
   TypeCache const& cache_;
+  int depthInFunction_;
```

Comment:
On Wed, Nov 4, 2015 at 3:27 PM, aseemgarg89 <notifications@github.com>
wrote:

> In src/wasm/asm-wasm-builder.cc
> <https://github.com/WebAssembly/v8-native-prototype/pull/50#discussion_r43956657>
> :
>
> > @@ -524,6 +527,7 @@ class AsmWasmBuilderImpl : public AstVisitor {
> >    Isolate* isolate_;
> >    Zone* zone_;
> >    TypeCache const& cache_;
> > +  int depthInFunction_;
>
> I was thinking depthInFunction_ and a stack of depthInLoop_ (after I
> implement loops, break, continue). Can also do block_depth and
> block_depth_in_loop.
>
You will probably need to keep a separate stack of loops to know exactly
which loop a JS continue or break statement targets.


> Btw, what is the convention around using camel case vs snake case?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WebAssembly/v8-native-prototype/pull/50/files#r43956657>
> .
>